stan/math/rev/scal/fun/inv_Phi.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef STAN_MATH_REV_SCAL_FUN_INV_PHI_HPP
+#define STAN_MATH_REV_SCAL_FUN_INV_PHI_HPP
+
+#include <stan/math/rev/core.hpp>
+#include <stan/math/prim/scal/fun/inv_Phi.hpp>
+#include <stan/math/prim/scal/fun/constants.hpp>
+
+namespace stan {
+  namespace math {
+
+    namespace {
+      class inv_Phi_vari : public op_v_vari {
+      private:
+        double m_z;
+      public:
+        explicit inv_Phi_vari(vari* avi) :
```

Comment:
No reason to store `m_z` here --- it'll be available as `val_` from the parent class